huaweicloud/services/elb/resource_huaweicloud_elb_active_standby_pool.go

```diff
@@ -0,0 +1,666 @@
+							}, false),
+						},
+						"domain_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"expected_codes": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"max_retries_down": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"monitor_port": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"url_path": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"vpc_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"ip_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceActiveStandbyPoolCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	conf := meta.(*config.Config)
+	region := conf.GetRegion(d)
+
+	var (
+		createActiveStandbyPooltUrl    = "v3/{project_id}/elb/master-slave-pools"
+		createActiveStandbyPoolProduct = "elb"
+	)
+	elbClient, err := conf.NewServiceClient(createActiveStandbyPoolProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating ELB client: %s", err)
+	}
+	createActiveStandbyPoolPath := elbClient.Endpoint + createActiveStandbyPooltUrl
+	createActiveStandbyPoolPath = strings.ReplaceAll(createActiveStandbyPoolPath, "{project_id}",
+		elbClient.ProjectID)
+
+	createActiveStandbyPoolOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	createActiveStandbyPoolOpt.JSONBody = utils.RemoveNil(buildCreateActiveStandbyPoolBodyParams(d))
+	createActiveStandbyPoolResp, err := elbClient.Request("POST", createActiveStandbyPoolPath,
+		&createActiveStandbyPoolOpt)
+	if err != nil {
+		return diag.Errorf("error creating ELB active standby pool: %s", err)
+	}
+
+	createActiveStandbyPoolRespBody, err := utils.FlattenResponse(createActiveStandbyPoolResp)
+	if err != nil {
+		return diag.Errorf("error retrieving ELB active standby pool: %s", err)
+	}
+	id, err := jmespath.Search("pool.id", createActiveStandbyPoolRespBody)
+	if err != nil {
+		return diag.Errorf("error creating ELB active standby pool: ID is not found in API response")
+	}
+
+	d.SetId(id.(string))
+
```

Comment:
add wait logic like the resource of huaweicloud_elb_pool

Reply:
done